lib/tasks/serve.js

```diff
@@ -26,6 +27,7 @@ class ServeTask extends Task {
       outputPath: options.outputPath,
       project: this.project,
       environment: options.environment,
+      broccoliMaster: experiments.BROCCOLI_MASTER,
```

Comment:
Should be able to revert changes to this file once the builder itself does its own experiments check...